parsing/parse_hrp.py

```diff
@@ -24,16 +24,18 @@ def parse_hrperf_log(perf_log_path, max_frequency_ghz):
             if len(data) < entry_size:
                 break
 
-
             # Unpack the data according to the updated structure
             cpu_id, ktime, stall_mem, inst_retire, cpu_unhalt, llc_misses, sw_prefetch = struct.unpack(entry_format, data)
 
+
+            timestamp_ns = int(ktime / tsc_per_us * 1e3)
```

Comment:
Make it conditionally based on the provided flag, so that the parsing script can work in both cases.